### PR TITLE
[13.0][MIG] stock_picking_back2draft

### PR DESCRIPTION
**Pull request by ps-tubtim**

### CODE REVIEW

File: stock_picking_back2draft/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# © 2016 Lorenzo Battistini - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Pickings back to draft",
+    "summary": "Reopen cancelled pickings",
+    "version": "13.0.1.0.0",
+    "category": "Warehouse Management",
+    "website": "https://www.agilebg.com",
```
Comment (acsonefho) [inline stock_picking_back2draft/__manifest__.py]: It should be:
`"website": "https://github.com/OCA/stock-logistics-workflow",`

File: stock_picking_back2draft/models/stock.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# © 2016 Lorenzo Battistini - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, models
+from odoo.exceptions import UserError
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
```
Comment (acsonefho) [inline stock_picking_back2draft/models/stock.py]: Can you split into 2 files please? 1 file per `Model`

**Review by acsonefho — CHANGES_REQUESTED** [review]
Tested and works properly.
Just few minor changes.
Thanks for the migration!

**Comment by ps-tubtim** [comment]
@acsonefho Thank you for review.

**Review by CasVissers-360ERP — APPROVED** [review]
Functional review

**Review by acsonefho — APPROVED** [review]

**Pull request merged**